lib/sdam/server.js

```diff
@@ -511,7 +511,11 @@ function makeOperationHandler(server, connection, cmd, options, callback) {
           session.serverSession.isDirty = true;
         }
 
-        if (supportsRetryableWrites(server) && !inActiveTransaction(session, cmd)) {
+        if (
+          isRetryableWriteError(err) &&
```

Comment:
Should this check if the error is `RetryableWriteError` before adding the label?